kolibri/core/assets/src/apiResources/contentMetaData.js

```diff
@@ -0,0 +1,15 @@
+const Resource = require('../api_resource').Resource;
+
+class ContentMetaDataResource extends Resource {
```

Comment:
I would have expected this class to set the super `Resource`'s API `name` and `idKey`, since those are the constant attributes that characterize the API resource? I might be misunderstanding how these are intended to be used.